src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackComputeQuotaPluginTest.java

```diff
@@ -40,41 +40,56 @@ public void setUp() {
 		token.setAttributes(attributes);
 		token.setAccessId("");
 	}
-	
+
+	// test case: Tests if getTotalQuota(), getUsedQuota() and getAvailableQuota() returns the right quotas.
     @Test
     public void getUserQuotaTest() throws FogbowManagerException, UnexpectedException {
+	    // set up
         ComputeAllocation allocationTotal = new ComputeAllocation(64, 46080, 20);
```

Comment:
Where these magic numbers come from? Were they calculated by hand? We should think in a better way to generate this type of data.

Reply:
Now I reproduced the method where it is generated.